app/components/admin/edition_images/uploaded_images_component.html.erb

```diff
@@ -20,19 +20,39 @@
     in the document body.</p>
   <% end %>
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-one-third">
-      <img src="<%= lead_image[:url] %>" alt="<%= lead_image[:preview_alt_text] %>" class="app-view-edition-images__image">
-    </div>
-    <ul class="app-view-edition-images__details govuk-grid-column-two-thirds govuk-list govuk-list--bullet">
-      <li><strong>Caption: </strong><%= lead_image[:caption] %></li>
-      <li><strong>Alt text: </strong><%= lead_image[:alt_text] %></li>
-    </ul>
-    <div class="app-view-edition-images__actions govuk-grid-column-full govuk-button-group">
-      <% if @edition.type == "CaseStudy" %>
-      <% lead_image[:links].each do |link| %><%= link %><% end %>
+  <% if lead_image %>
```

Comment:
Is there a way to separate the logics between the blocks to show the lead image and the form? feels like it's a little bit difficult to read, but I get that div nestings could have an impact to how it's rendered...

e.g.
<govuk grid div>
[ if lead image exists >> show lead image block ]
Show the form
[ if case study then show case study specific text & buttons else normal buttons ] < this  logic can also be pulled into rb file?
[ if lead image exists >> show links ]